winterfell/webapp/src/js/claimConfirmCtrl.js

```diff
@@ -1,8 +1,98 @@
 var app = angular.module('vetafiApp');
-app.controller('claimConfirmCtrl', ['$scope', '$state', 'net', 'claimService', '$stateParams',
-  function($scope, $location, net, claimService, $stateParams) {
+app.controller('claimConfirmCtrl', ['$scope', '$state', 'net', 'claimService', '$stateParams', '$uibModal',
+  function($scope, $location, net, claimService, $stateParams, $uibModal) {
     $scope.claimId = $stateParams.claimId;
 
+    $scope.user = {
+      firstName: 'Aaron',
+      lastName: 'Hsu'
+    };
+
+    $scope.vaAddress = {
+      title: 'Department of Veteran Affairs',
+      address: {
+        line1: 'Somewhere Street',
+        city: 'Washington',
+        state: 'DC',
+        country: 'USA',
+        zip: '20854'
+      }
+    };
+
+    $scope.mailingList = [
+      $scope.vaAddress,
+      {
+        title: 'Home',
+        address: {
+          sendTo: 'Aaron Hsu',
+          line1: '1223 4th Street',
+          line2: 'Apt #211',
+          city: 'San Francisco',
+          state: 'CA',
+          country: 'USA',
+          zip: '94158'
+        }
+      },
+      {
+        title: "Brother's crib",
+        address: {
+          sendTo: 'Austin Hsu',
+          line1: '983 Victory Road',
+          city: 'Rockville',
+          state: 'MD',
+          country: 'USA',
+          zip: '20834'
+        }
+      },
+    ];
+    $scope.emailList = [
+      {
+        name: 'Me',
+        email: 'ahsu@gmail.com'
+      },
+      {
+        name: 'Mom',
+        email: 'mom@gmail.com'
+      }
+    ];
+
+    $scope.formsList = [
```

Comment:
I'm looking for forms that the user has filled out, not all forms. I don't think `formTemplateService` is what I'm looking for.